frontend/src/scenes/data-management/definition/DefinitionEdit.tsx

```diff
@@ -12,6 +12,7 @@ import { LemonTextArea } from 'lib/components/LemonTextArea/LemonTextArea'
 import { ObjectTags } from 'lib/components/ObjectTags/ObjectTags'
 import { isPostHogProp } from 'lib/components/PropertyKeyInfo'
 import { VerifiedEventCheckbox } from 'lib/components/DefinitionPopup/DefinitionPopupContents'
+import { LemonSelect } from 'lib/components/LemonSelect'
 
 export function DefinitionEdit(props: DefinitionEditLogicProps): JSX.Element {
```

Comment:
Should we show the property type on the `View` screen as well as allow editing it on `Edit` screen?

Reply:
Good question.

> @alexkim205 I chose not to display this in the screen, because it seems like we're re-using the same components for event and event properties?

I'm confused how this works, but for _now_ since what's viewed are event params, even on event properties, I decided not to add the property specific params.

Alex is probably going to be taking up some cleaning up tasks here, so I'll sweep this under (his) rug 🙈 . Unless ofcourse either of you feel strongly about fixing this here.

Reply:
Hmmm, yep, to unblock the specific user this is fine. Since we can point them at the edit screen. But it's not discoverable otherwise...

I guess we should add it to documentation too...

Reply:
https://posthog.com/docs/user-guides/events#:~:text=While%20ingesting%20events%20in%20the%20plugin%20server%20PostHog%20detects%20some%20type%20information%20for%20properties%20to%20allow%20more%20intelligent%20filtering.